sonar-plugin-api/src/main/java/org/sonar/api/internal/RuntimeApiVersionFactory.java

```diff
@@ -30,19 +30,19 @@
 /**
  * For internal use only.
  */
-public class SonarQubeVersionFactory {
+public class RuntimeApiVersionFactory {
 
   private static final String FILE_PATH = "/sq-version.txt";
 
-  private SonarQubeVersionFactory() {
+  private RuntimeApiVersionFactory() {
     // prevents instantiation
   }
 
-  public static SonarQubeVersion create(System2 system) {
+  public static SonarQubeVersion create(System2 system, boolean isSonarLint) {
     try {
       URL url = system.getResource(FILE_PATH);
       String versionInFile = Resources.toString(url, StandardCharsets.UTF_8);
-      return new SonarQubeVersion(Version.parse(versionInFile));
+      return new SonarQubeVersion(Version.parse(versionInFile), isSonarLint);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SonarQubeVersion"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)